spot-client/src/spot-tv/ui/components/meeting-frame/ZoomMeetingFrame/ZoomMeetingFrame.js

```diff
@@ -161,6 +172,17 @@ export class ZoomMeetingFrame extends AbstractMeetingFrame {
             type
         });
 
+        if (typeof type === 'object') {
+            // This is a native controller command that has type and data wrapped
```

Comment:
looks like the code in `ZoomMeetingFrame` stayed, but it's in the abstract class already?